### DISCUSSION TITLE
Failed to launch browser: Running as root without --no-sandbox is not supported.

### DISCUSSION THREAD

**Discussion by dylan-mariscal-smd** [getgauge/gauge#2194] [General]
![image](https://user-images.githubusercontent.com/72824554/150786562-ab40026f-47d2-43ed-9b7f-7ba20cb7e920.png)

**Comment by sriv** [comment]
I am guessing you are running gauge+js+taiko specs in docker? If so, you'll need to use the `--no-sandbox` flag for the browser. See https://docs.taiko.dev/taiko_in_docker/#a-note-on-the-flags-and-options for more details

**Reply by dylan-mariscal-smd** [reply]
Thanks for the answer! I'm not using a docker. How do I add the --no-sandbox flag with the terminal I'm on a server

**Reply by sriv** [reply]
you can set it using`TAIKO_ BROWSER_ARGS` env variable. See this as an example of how it's done in dockerfile: https://github.com/getgauge/template-js/blob/master/Dockerfile#L25
